src/System.Net.Requests/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
There are some Outerloop tests that create a HttpWebRequest and because of: https://github.com/dotnet/corefx/issues/16928 when running in parallel this tests would hang. e.g https://github.com/dotnet/corefx/blob/master/src/System.Net.Requests/tests/HttpWebResponseHeaderTest.cs#L24

I talked to @davidsh offline and we decided they should run without parallelization.